CHANGES.rst

```diff
@@ -122,6 +122,11 @@ resample_spec
 
 - Enable model metadata blending [#4765]
 
+  set_telescope_pointing
```

Comment:
Not sure, but sphinx might complain about the extra whitespace at the beginning of the line here (or the fact that that makes it extend past the end of the subsequent underline).